lib/CAPI/Dialect/OM.cpp

```diff
@@ -287,6 +288,26 @@ MlirAttribute omReferenceAttrGetInnerRef(MlirAttribute referenceAttr) {
       (Attribute)unwrap(referenceAttr).cast<ReferenceAttr>().getInnerRef());
 }
 
+//===----------------------------------------------------------------------===//
+// IntegerAttr API.
+//===----------------------------------------------------------------------===//
+
+bool omAttrIsAIntegerAttr(MlirAttribute attr) {
+  return unwrap(attr).isa<circt::om::IntegerAttr>();
+}
+
+MlirAttribute omIntegerAttrGetInt(MlirAttribute attr) {
+  return wrap(
+      (Attribute)unwrap(attr).cast<circt::om::IntegerAttr>().getValue());
```

Comment:
nit: I think MLIR style now prefers `cast<IntegerAttr>(unwrap(attr))`.

Also, is that `(Attribute)` cast really required?

Reply:
The cast was not required, and also updated the cast.